kubeflow/tf-serving/tf-serving.libsonnet

```diff
@@ -217,6 +224,16 @@
               if $.util.toBool($.params.deployHttpProxy) then
                 $.parts.httpProxyContainer,
             ],
+            volumes+: if 'modelStorageType' in $.params then 
```

Comment:
same here

Reply:
Reasonable. Yesterday I tried to make `modelStorageType` as a param for `ks generate tf-serving` but failed (so far). So I add `storageType` into `params`. When I use `modelStorageType` as a param, it shows some error (maybe meets some conflicts).

Reply:
I mean, just doing

params:: {
  modelStorageType: "cloud",
}

It will get overridden if you pass the param modelStorageType